### ISSUE TITLE
Broadcom DUT lost configuration after reset_factory/test_reset_factory.py

### ISSUE THREAD

**Issue by ZhaohuiS** [sonic-net/sonic-mgmt#11103]
**Description**

**Steps to reproduce the issue:**
1. run reset_factory/test_reset_factory.py on master image
2. check DUT status
3.

**Describe the results you received:**

admin@sonic:~$ show version

SONiC Software Version: SONiC.master.436895-2ba3c90ad
SONiC OS Version: 12
Distribution: Debian 12.2
Kernel: 6.1.0-11-2-amd64
Build commit: 2ba3c90ad
Build date: Tue Dec 19 12:38:06 UTC 2023
Built by: AzDevOps@vmss-soni002OXR

Platform: x86_64-dell_s6100_c2538-r0
HwSKU: Force10-S6100
ASIC: broadcom
ASIC Count: 1
Serial Number: N/A
Model Number: N/A
Hardware Revision: N/A
Uptime: 06:31:26 up 3 min,  1 user,  load average: 3.66, 2.55, 1.08
Date: Wed 20 Dec 2023 06:31:26

Docker images:
REPOSITORY                    TAG                       IMAGE ID       SIZE
docker-syncd-brcm             latest                    2e2323798dff   716MB
docker-syncd-brcm             master.436895-2ba3c90ad   2e2323798dff   716MB
docker-gbsyncd-broncos        latest                    055551efbc5a   353MB
docker-gbsyncd-broncos        master.436895-2ba3c90ad   055551efbc5a   353MB
docker-gbsyncd-credo          latest                    626f16363cce   325MB
docker-gbsyncd-credo          master.436895-2ba3c90ad   626f16363cce   325MB
docker-dhcp-relay             latest                    123cea7fe661   311MB
docker-macsec                 latest                    fcdf7862062e   331MB
docker-eventd                 latest                    becd11c5398b   302MB
docker-eventd                 master.436895-2ba3c90ad   becd11c5398b   302MB
docker-orchagent              latest                    1272c248d858   340MB
docker-orchagent              master.436895-2ba3c90ad   1272c248d858   340MB
docker-fpm-frr                latest                    b66749716668   360MB

admin@sonic:~$ show ip interface
Interface    Master    IPv4 address/mask    Admin/Oper    BGP Neighbor    Neighbor IP
-----------  --------  -------------------  ------------  --------------  -------------
Loopback0              10.1.0.1/32          up/up         N/A             N/A
docker0                240.127.1.1/24       up/down       N/A             N/A
lo                     127.0.0.1/16         up/up         N/A             N/A
admin@sonic:~$ docker ps
CONTAINER ID   IMAGE                             COMMAND                  CREATED         STATUS          PORTS     NAMES
149eff4fddd7   docker-router-advertiser:latest   "/usr/bin/docker-ini…"   3 minutes ago   Up 45 seconds             radv
30c720869fa5   docker-teamd:latest               "/usr/local/bin/supe…"   3 minutes ago   Up 41 seconds             teamd
f890d5eed701   docker-fpm-frr:latest             "/usr/bin/docker_ini…"   3 minutes ago   Up 43 seconds             bgp
514f4aeb2139   docker-orchagent:latest           "/usr/bin/docker-ini…"   3 minutes ago   Up 49 seconds             swss
0368b52a6cb7   docker-eventd:latest              "/usr/local/bin/supe…"   3 minutes ago   Up 3 minutes              eventd
48996ff9b8ad   docker-database:latest            "/usr/local/bin/dock…"   3 hours ago     Up 3 minutes              database
admin@sonic:~$ show interface status
  Interface    Lanes    Speed    MTU    FEC    Alias    Vlan    Oper    Admin    Type    Asym PFC
-----------  -------  -------  -----  -----  -------  ------  ------  -------  ------  ----------
admin@sonic:~$ 
**Describe the results you expected:**

**Additional information you deem important:**

    **Output of `show version`:**

    ```
    (paste your output here)
    ```

    **Attach debug file `sudo generate_dump`:**

    ```
    (paste your output here)
    ```

**Comment by auspham** [comment]
Hi @ZhaohuiS  do you happen to know the latest status of this test case here? 

Also is Broadcom the only platform that's affected?

**Comment by ZhaohuiS** [comment]
it turned out it failed on many platforms, not only Broadcom.

<img width="1379" alt="Image" src="https://github.com/user-attachments/assets/731e3585-edc0-4354-9d80-c17f2558fef9" />

**Comment by yanaport** [comment]
Hi @ZhaohuiS,

Reset factory is supposed to reset configuration to factory default as designed, including login credentials. As a result, it will be impossible to connect the dut using non-default user/password.

Can you please elaborate on the issue you see? What is lost, switch configuration or dut connectivity?

**Comment by ZhaohuiS** [comment]
@yanaport Yes, DUT lost everything, we are unable to login device after this case.
I agree with you about the functionality of the test case, but could you please make sure configuration  can be restored after this case, to make sure it will not impact the test of other case, don't leave the DUT unreachable, which is not acceptable.
